### PR TITLE
Revert "Revert "Anonymous Web Apps usage""

### PR DESCRIPTION
**Pull request by benrudolph**
Reverts dimagi/commcare-hq#14997

### CODE REVIEW

**Comment by benrudolph** [comment]
https://github.com/dimagi/formplayer/pull/292

File: corehq/couchapps/hash_to_anonymous_app/views/view/map.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+function(doc){
```
Comment (dimagimon) [inline corehq/couchapps/hash_to_anonymous_app/views/view/map.js]: Parsing error: Unexpected token (

**Comment by orangejenny** [comment]
Hey Ben, fyi this had merge conflicts in `corehq/apps/cloudcare/templates/preview_app/base.html` and I resolved it by accepting the head changes.

**Comment by nickpell** [comment]
merged master (had conflicts)

**Comment by benrudolph** [comment]
@snopoke 
finally got a chance to take a look at this on staging. it's up and running:

screen shot 2017-03-01 at 3 01 29 pm

i checked to ensure all the commits were on staging and indeed:

cchq@hqdb1-staging:~/www/staging/current$ git log | grep 8e757c5
Merge: e5c3553 8e757c5
commit 8e757c5cf178c5d7b1f2c7b357938cbf4729bce4

no crazy template errors anymore, and i also ran both the formplayer and hq branch locally to check if i could use app preview and was able to do so without error. i think this is ready again. when this gets merged i'm going to deploy it myself to make sure everything is fine (there are a lot of moving parts)

**Pull request merged**